util/types/mydecimal.go

```diff
@@ -690,6 +700,10 @@ func (d *MyDecimal) Round(to *MyDecimal, frac int) (err error) {
 	wordsInt := digitsToWords(int(d.digitsInt))
 
 	var roundDigit int32 = 5
+	/* TODO - fix this code as it won't work for CEILING mode */
```

Comment:
To support CEILING mode in the future, switch round mode to set the roundDigit, then switch roundDigit at line 743 is the right way.